Sources/MapboxNavigation/BannerPreviewing.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+// :nodoc:
+public protocol BannerPreviewing: UIViewController {
+    
+    var configuration: PreviewBannerConfiguration { get }
+}
+
+// :nodoc:
+public struct PreviewBannerConfiguration {
+    
+    // :nodoc:
+    public var position: Banner.Position = .bottomLeading
+    
+    // :nodoc:
+    public var height: CGFloat? = nil
```

Comment:
Every banner that is presented in `PreviewViewController` must conform to `BannerPreviewing` protocol and provide default configuration via `PreviewBannerConfiguration`. I'm not yet sure if `height` property will remain here though. It'd be good to have the ability to set [detents](https://developer.apple.com/documentation/uikit/uisheetpresentationcontroller/3801903-detents) instead of plain height of the banner that can only be extended/collapsed. In any case this additional functionality will be added in other PR.